app/views/users/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="well panel panel-default">
+  <div class="panel-body">
+    <div class="row">
+      <div class="col-xs-12 col-sm-4 text-center">
+        <h2><%= @user.username %></h2>
+      </div>
+      <!--/col-->
+      <div class="col-xs-12 col-sm-6">
+        <p><strong>Email</strong>: <%= @user.email %></p>
+        <p><strong>Courses:</strong></p>
+        <ul>
+          <% @courses.each do |course| %>
+          <li><%= link_to course, course %>
+            <% course.roles.where(user: @user).each do |role| %>
```

Comment:
Please do not write queries in the views. The more we use that whole `Role`s table the less I like it.